x-pack/plugins/apm/server/routes/typings.ts

```diff
@@ -39,6 +39,7 @@ export interface APMRouteCreateOptions {
       | 'access:ml:canGetJobs'
       | 'access:ml:canCreateJob'
     >;
+    body?: { accepts: Array<'application/json' | 'multipart/form-data'> };
```

Comment:
The header is called [`Accept`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept), so should this also be singular to match that?

Reply:
it actually doesn't work if I change it to `accept`